testing_and_setup/compass/ocean/drying_slope/marsh_flooding/100m/movie.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
```

Comment:
Same comment -- can we make code more modular so we don't have copy/pasted script?  Not a big deal, but makes the repo nicer because if there is a bug, only have to fix it in a single place.  It also keeps code shorter too.